src/XGBoostSharp.Tests/DMatrixTest.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using XGBoostSharp.lib;
+
+namespace XGBoostSharp.Test;
+
+[TestClass]
+public class DMatrixTest
+{
+    readonly float[][] m_dataTrain = [[1, 2, 3, 4]];
+    readonly float[] m_labelsTrain = [1];
+
+    [TestMethod]
+    public void DMatrix_SetFeatureNames()
+    {
+        string[] featureNames = ["f1", "f2", "f3", "f4"];
+
+        var sut = new DMatrix(m_dataTrain, m_labelsTrain);
```

Comment:
@madskonradsen Remember using statement for the DMatrix:
```csharp
using var sut = new DMatrix(m_dataTrain, m_labelsTrain);
```
Same for the the other tests.